demo/b2share_demo/helpers.py

```diff
@@ -328,12 +344,20 @@ def download_v1_record(directory, record, verbose=False):
                 click.secho('    Download file "{}"'.format(file_dict.get('name')))
             filepath = os.path.join(directory, 'file_{}'.format(index))
             _save_file(file_dict['url'], filepath)
-            if int(os.path.getsize(filepath)) != int(file_dict.get('size')):
-                raise Exception("downloaded file size differs, {}".format(filepath))
+            #if int(os.path.getsize(filepath)) != int(file_dict.get('size')):
```

Comment:
But we must make sure that the sizes are the same! It must be exactly the same data.